static/js/emoji_picker.js

```diff
@@ -241,6 +240,20 @@ function filter_emojis() {
                         break; // We only need the first matching alias per emoji.
                     }
                 }
+
+                const emoji_info = emoji.get_emoji_details_by_name(emoji_dict.name);
+                if (typeahead.is_unicode_emoji(emoji_info)) {
+                    // convert emoji_dict to an actual emoji character
+                    const parsed_emoji_code = typeahead.parse_unicode_emoji_code(
+                        emoji_info.emoji_code,
+                    );
+                    // using query instead of search_terms because it's possible multiple emojis were input
+                    // without being separated by spaces
```

Comment:
Can you extend the tests to include one that covers this possibility?  You can hand-verify that the test is right -- it should fail if we change this line to `search_results.includes`.

Reply:
I couldn't find any test for the `filter_emojis` function so far (I think there isn't any for it). I would have loved to add tests for it, but I'm having a hard time wrapping my head around how to go about adding tests for functions that require `$("...")` dom manipulation functionality.

So what I've done instead is I've pulled the emoji unicode matching logic into a function named `is_emoji_present_in_text`, and I've added test for that instead. Please let me know what you think.

Reply:
That seems like a reasonable plan.